src/ui/widgets/chlcc/titlebutton.cpp

```diff
@@ -0,0 +1,52 @@
+#include "titlebutton.h"
+
+#include "../../../renderer2d.h"
+#include "../../../profile/games/chlcc/titlemenu.h"
+
+namespace Impacto {
+namespace UI {
+namespace Widgets {
+namespace CHLCC {
+
+using namespace Impacto::Profile::CHLCC::TitleMenu;
+
+void TitleButton::Render() {
+  glm::vec4 black(0.0f);
+  black.a = Opacity;
+  glm::vec4 white(1.0f);
+  white.a = Opacity;
+
+  if (HasFocus) {
+    if (!isSubButton) {  //Main buttons
+    Renderer2D::DrawSprite(HighlightSprite,
+                           glm::vec2(Bounds.X - ItemHighlightOffsetX,
+                                     Bounds.Y - ItemHighlightOffsetY),
+                           white);
+    Renderer2D::DrawSprite(FocusedSprite,
+                           glm::vec2(Bounds.X, Bounds.Y),
+                           white);
+    } else {  //Sub buttons
+      Renderer2D::DrawSprite(
+          HighlightSprite,
+          glm::vec2(SecondaryItemHighlightX, Bounds.Y - ItemHighlightOffsetY),
```

Comment:
Maybe it would be better to make this a relative position, that's what I've been trying to do in widgets.